index.html

```diff
@@ -124,5 +124,30 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-and-aearly-access-section">
+    <div id="join-hiring" class="card-subsection">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
```

Comment:
This is lacking the link. It also lacks the target=blank attribute that makes this link open up in a new tab. Since it opens in a new tab, it's a link to an external resource. 

Putting this sort of placeholder doesn't let me know if you know how to handle those or if you take measures to do so.

Reply:
> No entiendo a que te refieres con "Putting this sort of placeholder". Y supongo que el link que te refieres es al codigo de la review de uno mas arriba.

:v Los enlaces a sitios externos es mejor que sean absolutos y demás cosas. Si nisiquiera pones un enlace y dejas un ```<a href="#"></a>``` no podría saber si sabías o que weas. Si se te pasaba por alto y tenías que arreglarlo luego, es otro pull request más que te hubieras ahorrado solo poniendo el enlace que se usa ahí y no un relleno que no hace nada.

Reply:
Como es un enlace a un sitio externo no absoluto?
Que sea un sitio externo a nuestra pagina ya implica que deba ser absoluto, o no?

Reply:
> Como es un enlace a un sitio externo no absoluto? Que sea un sitio externo a nuestra pagina ya implica que deba ser absoluto, o no?

xdn't uno ve de todo cuando se asoma la gente a codear ajskajks. Lo voy a dar por resuelto entonces.